bash_completion

```diff
@@ -3,19 +3,19 @@
 # bash completion for Node Version Manager (NVM)
 
 if ! command -v nvm &> /dev/null; then
-    return
```

Comment:
is editorconfig not being enforced on this file already?

Reply:
Nope as it can't understand the syntax, it can yell when the indentation is 3 or 5 spaces, but as we set to 2 space, 4 or 6 spaces will be both fine to it.

Reply:
Why can't it understand the syntax?

Reply:
it's not language based.